diff --git a/node_modules/@stoplight/elements-dev-portal/index.esm.js b/node_modules/@stoplight/elements-dev-portal/index.esm.js
index 14d5178..72a2362 100644
--- a/node_modules/@stoplight/elements-dev-portal/index.esm.js
+++ b/node_modules/@stoplight/elements-dev-portal/index.esm.js
@@ -158,11 +158,19 @@ function __awaiter(thisArg, _arguments, P, generator) {
     });
 }
 
+const filterToc = (toc, page) => {
+  if (page == 'Guides') {
+    return toc.filter((x) => x.title == 'apps');
+  } else if (page == 'Concepts') {
+    return toc.filter((x) => x.title == 'getting-started');
+  } else return toc;
+};
+
 const TableOfContents = (_a) => {
-    var { tableOfContents, activeId, Link, collapseTableOfContents = false, externalScrollbar, onLinkClick } = _a, boxProps = __rest(_a, ["tableOfContents", "activeId", "Link", "collapseTableOfContents", "externalScrollbar", "onLinkClick"]);
+    var { tableOfContents, activeId, Link, collapseTableOfContents = false, externalScrollbar, onLinkClick, page } = _a, boxProps = __rest(_a, ["tableOfContents", "activeId", "Link", "collapseTableOfContents", "externalScrollbar", "onLinkClick"]);
     return (React.createElement(Flex, Object.assign({ bg: "canvas-100" }, boxProps, { flexDirection: "col", maxH: "full" }),
         React.createElement(Flex, { flexGrow: true, flexShrink: true, overflowY: "auto" },
-            React.createElement(TableOfContents$1, { tree: tableOfContents.items, activeId: activeId, Link: Link, maxDepthOpenByDefault: collapseTableOfContents ? 0 : 1, externalScrollbar: externalScrollbar, onLinkClick: onLinkClick })),
+            React.createElement(TableOfContents$1, { tree: filterToc(tableOfContents.items, page), activeId: activeId, Link: Link, maxDepthOpenByDefault: collapseTableOfContents ? 0 : 1, externalScrollbar: externalScrollbar, onLinkClick: onLinkClick })),
         tableOfContents.hide_powered_by ? null : (React.createElement(PoweredByLink, { source: activeId, pathname: typeof window !== 'undefined' ? window.location.pathname : '', packageType: "elements-dev-portal" }))));
 };
 
